features/FxA-46-sms-install-link/README.md

```diff
@@ -0,0 +1,61 @@
+# SMS Install Link
+
+https://mozilla.aha.io/features/FXA-46
+
+## Stories
+
+As a user signing up for a Firefox Account, I may want to easily install
+Firefox on my Android or iOS device, removing the friction of searching
+by sending me the link in a text message.
+
+## Success Criteria
+
+This feature will be successful if we measurably increase the rate of
+Firefox activations for Android and iOS.
+
+A metrics funnel we can use to measure success is:
+
+1. Number of users who submit their number to the form in order to
+   receive a SMS
+2. Number of users who click the link
+3. Number of users who install Firefox from this link (utm_campaign
+   thingies?)
+4. Number of users who connect a second device after having requested a
+   link
```

Comment:
ISTM we can measure 1, 2 and 4 ourselves, but 3 we'll have to get from someone else.  Growth team may be able to help us out here, but we can also push ahead with *just* 1, 2 and 4 if we need to.